.pre-commit-config.yaml

```diff
@@ -19,3 +19,8 @@ repos:
   rev: 22.3.0
   hooks:
     - id: black
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v1.3.0
+  hooks:
+  -   id: mypy
+      exclude: 'test_.*?\.py$'
```

Comment:
What does it do here to ignore the test files?

Reply:
Yes, ignores the test files which (in this project) sit together with the others.
Some tests use incorrect types on purpose..